### PR TITLE
feat: allow game to specify max atlas sizes

### PR DESCRIPTION
**Pull request by erickzanardo**
FlameTiled allows us to specify what are the max dimensions of the atlas used to pack the images in a map, but there were no way to specify them when using Leap helper to load map, this PR address that.

### CODE REVIEW

File: packages/leap/lib/src/leap_map.dart [hunk 1]
```diff
@@ -120,6 +120,8 @@ class LeapMap extends PositionComponent with HasGameRef<LeapGame> {
       prefix: prefix,
       bundle: bundle,
       images: images,
```
Comment (kurtome) [inline packages/leap/lib/src/leap_map.dart:122]: I wonder if `prefix`, `bundle`, and `images` should also be moved to `LeapTiledOptions`

**Review by kurtome — APPROVED** [review]

File: packages/leap/lib/src/leap_map.dart (hunk 1 above)
Comment (erickzanardo) [reply packages/leap/lib/src/leap_map.dart:122]: yeah, that might be a good idea! Those are only used while loading the tiled map iirc.

Should we merge this, then open an issue to track this other change?

Comment (kurtome) [reply packages/leap/lib/src/leap_map.dart:122]: Sounds good! I mostly just want to keep the params from getting crazy here

Do you have permissions to merge? Feel free if so, not sure how if I need to add you or what

Comment (erickzanardo) [reply packages/leap/lib/src/leap_map.dart:122]: @kurtome write permission in the repo is needed for me to merge the PR, I've been doing all these PRs from my fork :D

**Pull request merged**